pkg/output/writer.go

```diff
@@ -0,0 +1,10 @@
+package output
+
+import (
+	"github.com/thanos-community/obslytics/pkg/dataframe"
+)
+
+type Writer interface {
+	Write(dataframe.Dataframe) error
```

Comment:
this is meant to be parquet data frame or just generic one?

Reply:
This would be a generic one: the idea is the ingestor is format agnostic and just prepare the data in tabular ready format, that each writer can understand and produce the specific format for it